prac_09/taxi_simulator.py

```diff
@@ -0,0 +1,75 @@
+"""
+Write a taxi simulator program that uses the Taxi and SilverServiceTaxi classes.
+"""
+from prac_09.taxi import Taxi
+from prac_09.silver_service_taxi import SilverServiceTaxi
+
+MENU = """q)uit, c)hoose taxi, d)rive"""
+
+
+def main():
+    """Each time, until they quit: user can choose a taxi and drive a desired distance.
+    At the end of each trip, the trip cost is displayed and add it to their total bill."""
+    taxis = [Taxi("Prius", 100), SilverServiceTaxi("Limo", 100, 2), SilverServiceTaxi("Hummer", 200, 4)]
+    current_taxi = None
```

Comment:
Very nice - you avoided the use of global variables, which are commonly known to be evil.